src/moscot/utils/subset_policy.py

```diff
@@ -51,7 +51,21 @@ def _format(self, value: Any, *, is_source: bool) -> Any:
 
 
 class SubsetPolicy(Generic[K], abc.ABC):
-    """Policy class."""
+    r"""Base policy class.
+
+    Parameters
+    ----------
+    adata
+        Annotated data object or a categorical data.
+    key
+        Key in :attr:`~anndata.AnnData.obs` where the categorical data is stored.
+    verify_integrity
+        Whether to check that the data has :math:`\ge 2` categories.
+
+    Examples
+    --------
+    TODO(michalk8): add usage example
```

Comment:
@ArinaDanilina please open issue and self-assign

Reply:
#562